bgpd/bgp_nht.c

```diff
@@ -911,3 +911,52 @@ void bgp_nht_register_enhe_capability_interfaces(struct peer *peer)
 						BGP_UNNUM_DEFAULT_RA_INTERVAL);
 	}
 }
+
+void bgp_nht_dereg_enhe_capability_intfs(struct peer *peer)
+{
+	struct bgp *bgp;
+	struct bgp_node *rn;
+	struct bgp_nexthop_cache *bnc;
+	struct nexthop *nhop;
+	struct interface *ifp;
+	struct prefix p;
+
+	if (peer->ifp)
+		return;
+
+	bgp = peer->bgp;
+
+	if (!bgp->nexthop_cache_table[AFI_IP6])
+		return;
+
+	if (!sockunion2hostprefix(&peer->su, &p)) {
+		if (BGP_DEBUG(nht, NHT))
+			zlog_debug("%s: Unable to convert prefix to sockunion",
```

Comment:
isn't this sockunion to prefix convertion?

Reply:
I'm sorry, but I don't understand the question.  Please explain.

Reply:
sockunion2hostprefix means `sockunion to hostprefix`? :) But zlog_debug says: `prefix to sockunion`.

Reply:
Got it.  Thanks for explaining.  I'll change the debug output.